### DISCUSSION TITLE
Remove Armory Crate Popup when pushing M4 button

### DISCUSSION THREAD

**Discussion by bohnanza22** [seerge/g-helper#239] [General]
I didn't see a discussion on this yet so I figured I would ask. 

I have ran the debloat script and uninstalled the Armory Crate Button application in Add Remove Programs but I am still getting this popup.  Anyone know how to disable this (see screenshot)? 

<img width="980" alt="2023-04-13_10-01-15 - Copy" src="https://user-images.githubusercontent.com/110491105/231832695-1eb13e8e-6b6b-4036-83e4-55a1da1e646e.png">

**Comment by seerge** [comment]
@bohnanza22 debloat script has nothing to do with a popup. Your question is literally addressed in a very first FAQ item
https://github.com/seerge/g-helper#how-do-i-stop-the-armory-crate-install-popup-appearing-every-time-i-press-the-m4--rog-key

**Reply by bohnanza22** [reply]
Ahh thanks for pointing that out to me. This was like not finding what you are looking for in the fridge. Until you realize it's right there in front of your face.  Thanks!
